public/index.html

```diff
@@ -7,8 +7,8 @@
     <meta name="description" content="Avalanche Explorer is an analytics tool that enables people to search the Avalanche blockchain for transactions, addresses, and other platform activities." vue-router-data="">
     <meta property="og:description" content="Avalanche Explorer is an analytics tool that enables people to search the Avalanche blockchain for transactions, addresses, and other platform activities." vue-router-data="">
     <meta property="og:image" content="<%= BASE_URL %>og-image-avalanche.png">
-    <meta property="og:url" content="https://explorer.avax.network/">
-    <meta property="og:type" content="website">    
+    <meta property="og:url" content="https://explorer-xp.avax.network/">
```

Comment:
Nice addition, but this is going to just redirect to `subnets.avax.network`.  Can we set this to `https://explorer-xp.avax.network/subnets` so that metadata actually opens the XP explorer?

Reply:
Good call! Making that change now.